src/assets/translations/en.js

```diff
@@ -28,10 +28,8 @@ export default {
     hd: 'Quality',
     liveBroadcast: 'Live',
     logo: 'Logo',
-    more: 'More',
     next: 'Next',
     nextUp: 'Next Up',
-    nextUpClose: 'Next Up Close',
```

Comment:
It's not enough to remove this. It should be a fallback for `localization.close`. So in the backwards compatibility mapping:
`localization.close = localization.close || localization. nextUpClose;`

Reply:
Let's address once (or in) https://github.com/jwplayer/jwplayer/pull/3084 is